app/views/capacity.html

```diff
@@ -40,6 +40,71 @@
 
 <div id="plotly-div-line" role="presentation"></div>
 
+{% if organisationLevel !== 'offender-manager' %}
+  <h3 class="heading-medium">Capacity breakdown</h3>
+  <table id="example" class="data-table dataTable">
```

Comment:
`id=example` looks like a copy and paste typo.

Reply:
It sure is, fixed